Source/Extensions/Blazorise.Charts.DataLabels/wwwroot/chart.datalabels.js

```diff
@@ -52,7 +52,12 @@ function compileDatasetsOptionsCallbacks(options) {
 
     Object.keys(options).forEach(function (key) {
         if (options[key] && options[key].startsWith("function")) {
-            options[key] = parseFunction(options[key]);
+            if (key === 'scriptableFormatter') {
+                options['formatter'] = parseFunction(options[key]);
```

Comment:
Hacky way to make it backwards compatible and be able to keep the existing int rounding rules.